Sources/TransferMethod/SelectTransferMethodTypePresenter.swift

```diff
@@ -30,7 +30,7 @@ protocol SelectTransferMethodTypeView: class {
     func navigateToAddTransferMethodController(country: String,
                                                currency: String,
                                                profileType: String,
-                                               detail: TransferMethodTypeDetail)
+                                               transferMethodType: HyperwalletTransferMethodType)
```

Comment:
Update to transferMethodTypeCode: String

Reply:
fixed